File: lesson1/lesson1.cs
using System;

class Program
{
    static void Main()
    {
        // Prompt the user for the first number
        Console.Write("Enter the value for a: ");
        string inputA = Console.ReadLine();
        double a = Convert.ToDouble(inputA);

        // Prompt the user for the second number
        Console.Write("Enter the value for b: ");
        string inputB = Console.ReadLine();
        double b = Convert.ToDouble(inputB);

        // Add the numbers
        double c = a + b;

        // Print the result
        Console.WriteLine($"The result of a + b is: {c}");
    }
}

File: lesson1/lesson1.go
package main

import (
	"fmt"
)

func main() {
	var a, b float64

	// Get input from the user
	fmt.Print("Enter the value for a: ")
	fmt.Scanln(&a)

	fmt.Print("Enter the value for b: ")
	fmt.Scanln(&b)

	// Add the numbers
	c := a + b

	// Print the result
	fmt.Printf("The result of a + b is: %f\n", c)
}
